xrpl/helpers/converters_test.go

```diff
@@ -0,0 +1,37 @@
+package helpers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Is this package installed? If not, we can omit it and do comparisons simpler

Reply:
Yes it is installed and used in other tests.
What is the easier way to do the comparison ?